.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: "Release"
+
+on: workflow_dispatch
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: create branch and tag
+        run: |
+          OLD_VERSION=$(git describe --abbrev=0 --tags)
+          IFS=. read -r major minor patch <<< "$OLD_VERSION"
+          RELEASE_BRANCH="release-$major.$((++minor))" 
+          NEW_VERSION="$major.$((++minor)).0-rc1"
```

Comment:
oh wait sorry, this mutates `$minor` it seems

Reply:
so should be just `$minor` for the NEW_VERSION?

Reply:
get rid of the second increment, it seems working